modules/microshift-nw-multus-add-pod.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-nw-multus-add-pod_{context}"]
+= Adding a pod to an additional network
+
+You can add a pod to an additional network. At the time a pod is created, additional networks are attached to it. The pod continues to send normal cluster-related network traffic over the default network.
+
+If you want to attach additional networks to a pod that is already running, you must restart the pod.
+
+.Prerequisites
+
+* The OpenShift CLI (`oc`) is installed.
+* The cluster is running.
+* A network defined by a `NetworkAttachmentDefinition` object that you want to attach the pod to exists.
+
+.Procedure
+
+. Add an annotation to a `Pod` YAML file. Only one of the following annotation formats can be used:
+
+.. To attach an additional network without any customization, add an annotation with the following format. Replace `<network>` with the name of the additional network to associate with the pod:
++
+[source,yaml]
+----
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: <network>[,<network>,...] <1>
+----
+<1> Replace `<network>` with the name of the additional network to associate with the pod. To specify more than one additional network, separate each network with a comma. Do not include whitespace between the comma. If you specify the same additional network multiple times, that pod will have multiple network interfaces attached to that network.
++
+.Example annotation for a bridge-type additional network
++
+[source,yaml]
+----
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: bridge-conf
+----
+
+.. To attach an additional network with customizations, add an annotation with the following format:
++
+[source,yaml]
+----
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: |-
+      [
+        {
+          "name": "<network>", <1>
+          "namespace": "<namespace>", <2>
+          "default-route": ["<default-route>"] <3>
+        }
+      ]
+----
+<1> Specify the name of the additional network defined by a `NetworkAttachmentDefinition` object.
+<2> Specify the namespace where the `NetworkAttachmentDefinition` object is defined.
+<3> Optional: Specify an override for the default route, such as `192.168.17.1`.
+
+. To create a `Pod` YAML file and add the annotation for an additional network, run the following command and use the following example YAML:
```

Comment:
"the following example YAML" - is that the YAML in step 3? I don't see sample YAML in this step.

Reply:
Moved it up